Gemfile

```diff
@@ -158,7 +158,10 @@ end
 #   developers may not need or may not easily install, such as rails-dev-boost,
 #   any git based gem, and compiled gems like rbtrace or memprof.
 dev_gemfile = File.expand_path("Gemfile.dev.rb", __dir__)
-eval_gemfile(dev_gemfile) if File.exist?(dev_gemfile)
+if File.exist?(dev_gemfile)
+  Bundler::UI::Shell.new.warn "** /Gemfile.dev.rb deprecated, please move it to /bundler.d/"
```

Comment:
The leading slash is a bit odd, can it just be:

"** Gemfile.dev.rb deprecated, please move it to bundler.d/"